docs/tutorial-create-hello-world.md

```diff
@@ -1,3 +1,63 @@
 # Creating `hello-world.wasm`
 
-... more coming soon
+There are a number of ways to create `.wasm` files but for the purposes of this
+tutorial, we'll be using the Rust toolchain. You can find more information on
+creating `.wasm` files from other languages in the
+[Writing WebAssembly section](https://bytecodealliance.github.io/wasmtime/wasm.html).
```

Comment:
Oh for inter-links in the book it's probably best to use `wasm.md` instead of a full url to `wasm.html`, that way local renders should still work.